stable/traefik/values.yaml

```diff
@@ -111,3 +185,10 @@ deployment:
     httpEnabled: false
     httpsEnabled: false
     dashboardEnabled: false
+## Environment variables for the Traefik pod
+env:
+# Kubernetes ingress filters
+# kubernetes:
+#   namespaces:
+#   - default
+#   labelSelector:
```

Comment:
I don't quite understand how this section is being used.

Reply:
This is already on master as of 05bd3660

Reply:
Ok... I thought it looked familiar. Can I suggest rebasing on master then? I see there are complementary changes to this that are absent from the deployment in this PR-- the end result being that anything in this section won't ever be included.